colossalai/engine/schedule/_pipeline_schedule.py

```diff
@@ -115,7 +116,7 @@ def load_micro_batch(self):
     def pre_processing(self, engine):
         # TODO: remove this after testing new zero with pipeline parallelism
         model = engine.model
-        if isinstance(model, (NaiveAMPModel)) or hasattr(model, 'colo_attr'):
+        if isinstance(model, (NaiveAMPModel)) or isinstance(model, ShardedModelV2):
```

Comment:
revert it back to yuliang's logic. I mistakingly modify the code in the previous PR.